### PR TITLE
Add minted highlighter

### PR DESCRIPTION
**Pull request by l3kn**
As discussed in Issue #93,

I am not sure if more documentation or tests are needed.

### CODE REVIEW

File: doc/syntax_highlighter/minted.page [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Minted
+---
+
+## Syntax Highlighting With Minted
+
+[Rouge](https://github.com/gpoore/minted) (LPPL/BSD licensed)
```
Comment (gettalong) [inline doc/syntax_highlighter/minted.page]: Should Rouge be Minted?

**Comment by gettalong** [comment]
Looks good! However, it seems converting of code spans is somewhat supported but implemented?

**Comment by l3kn** [comment]
Added that, too.

**Comment by gettalong** [comment]
Thanks! I have merge your changes and added some fixes.

**Pull request closed without merging**